server/controllers/applications.coffee

```diff
@@ -424,3 +425,16 @@ module.exports =
                         success: true
                         msg: 'Application stopped'
                         app: req.application
+
+    fetchMarket: (req, res, next) ->
+        market.download (err, data) ->
+            if err?
+                res.send
+                    error: true
+                    success: false
+                    message: err
+                , 500
+            else
+                res.send
+                    success:true
+                    apps: data
```

Comment:
I suggest we stop sending "success: true" in the response. If the status code is 200, it's success. So if "data" is just an array, no need to send "apps: data":
```
res.send 200, data
```